.github/workflows/unit-tests.yml

```diff
@@ -55,6 +55,10 @@ jobs:
         run: |
           DEVICE=cpu make inference-api-e2e
 
+      - name: Run RAGEngine unit tests
+        run: |
+          make rag-service-test
+
```

Comment:
I think we are trying to split RAG engine and workspace instead of combining them.

Reply:
revert